variants/pure/pure.go

```diff
@@ -12,13 +12,12 @@ import (
 )
 
 var PureVariant = common.Variant{
-	Name:        "Pure",
-	Graph:       func() dip.Graph { return PureGraph() },
-	Start:       PureStart,
-	Blank:       PureBlank,
-	Phase:       classical.Phase,
-	ParseOrders: orders.ParseAll,
-	ParseOrder:  orders.Parse,
+	Name:   "Pure",
+	Graph:  func() dip.Graph { return PureGraph() },
+	Start:  PureStart,
+	Blank:  PureBlank,
+	Phase:  classical.Phase,
+	Parser: orders.ClassicalParser,
 	OrderTypes: []dip.OrderType{
```

Comment:
This can be tidied up now too - we could have a PureParser (that misses the convoy orders), rather than using the ClassicalParser.

Reply:
Nice!